tests/core/test_actor.py

```diff
@@ -43,3 +46,45 @@ def test_backends_with_colliding_uri_schemes_fails(self):
 
     def test_version(self):
         self.assertEqual(self.core.version, versioning.get_version())
+
+
+class CoreActorExportRestoreTest(unittest.TestCase):
+
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+        config = {
+            'core': {
+                'max_tracklist_length': 10000,
+                'restore_state': True,
+                'data_dir': self.temp_dir,
+            }
+        }
+
+        os.mkdir(os.path.join(self.temp_dir, 'core'))
+
+        self.core = Core.start(
+            config=config, mixer=None, backends=[]).proxy()
+
+    def tearDown(self):  # noqa: N802
+        pykka.ActorRegistry.stop_all()
+        shutil.rmtree(self.temp_dir)
+
+    def test_export_state(self):
+        self.core.teardown().get()
+        # TODO: implement meaningful test
```

Comment:
At the very least, load the saved file and make a few assertions on its content.